admin-portal/src/config.ts

```diff
@@ -1,9 +1,13 @@
-export default {
+import { AuthReactConfig, Storage } from "@asgardeo/auth-react";
+import { STSClientConfig } from "@asgardeo/token-exchange-plugin";
+
+const SDKConfig: AuthReactConfig & STSClientConfig = {
   clientID: "W_ZaTLGR20OKy0HTYPqSAJvuS5Ia",
   baseUrl: "https://api.asgardeo.io/t/elixir",
   signInRedirectURL: `${process.env.REACT_APP_SIGN_IN_REDIRECT_URL}`,
   signOutRedirectURL: `${process.env.REACT_APP_SIGN_OUT_REDIRECT_URL}`,
   scope: ["openid", "email", "profile"],
+  storage: Storage.WebWorker,
```

Comment:
In order to use webworker as the storage mechanism, this should be addressed beforehand IMO https://github.com/asgardeo/asgardeo-auth-react-sdk/issues/120

Reply:
Let's revert this until then.

Reply:
Fixed with https://github.com/LSFLK/MedicinesforLK/pull/430/commits/4759273784f0223c547229f90457537142cdfa32.